Leetcode/tree/binary_tree.hpp

```diff
@@ -0,0 +1,68 @@
+//
+//  binary_tree.hpp
+//  tree
+//
+//  Created by dsh on 2021/7/21.
+//
+
+#ifndef TREE_BINARY_TREE_HPP
+#define TREE_BINARY_TREE_HPP
+
+#include <vector>
+
+struct TreeNode {
+  public:
+    explicit TreeNode() { }
```

Comment:
```suggestion
    explicit TreeNode() {}
```

规范里面全是 `{}`，搜不到 `{ }`。